apps/framework-cli/src/project.rs

```diff
@@ -502,3 +536,55 @@ impl Project {
         None
     }
 }
+
+// Tests
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+
+    fn create_python_project() -> Project {
+        Project::new(
+            Path::new("tests/python/project"),
+            "test_project".to_string(),
+            SupportedLanguages::Python,
+        )
+    }
+
+    fn remove_python_project() {
```

Comment:
Same as above - for those tests we should probably use `assert_fs` it will create isolated folders and gas all the setup and tear down logic in there